homeassistant/components/ping/config_flow.py

```diff
@@ -0,0 +1,99 @@
+"""Config flow for Ping (ICMP) integration."""
+from __future__ import annotations
+
+from collections.abc import Callable, Coroutine, Mapping
+import logging
+from typing import Any, cast
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST, CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import selector
+from homeassistant.helpers.schema_config_entry_flow import (
+    SchemaCommonFlowHandler,
+    SchemaConfigFlowHandler,
+    SchemaFlowFormStep,
+    SchemaFlowMenuStep,
+)
+
+from .const import CONF_PING_COUNT, DEFAULT_PING_COUNT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def choose_options_step(options: dict[str, Any]) -> str:
+    """Return next step_id for options flow according to platform."""
+    return cast(str, options["platform_type"])
+
+
+def set_platform(
+    platform: str,
+) -> Callable[
+    [SchemaCommonFlowHandler, dict[str, Any]], Coroutine[Any, Any, dict[str, Any]]
+]:
+    """Set platform type."""
+
+    async def _set_platform(
+        handler: SchemaCommonFlowHandler, user_input: dict[str, Any]
+    ) -> dict[str, Any]:
+        """Add platform type to user input."""
+        return {"platform_type": platform, **user_input}
+
+    return _set_platform
+
+
+BINARY_SENSOR_OPTIONS_SCHEMA = vol.Schema(
```

Comment:
It would be great if the update_interval/polling_interval can also be configured via the UI.

Reply:
We don't want that to be configured via UI. If you want to change the amount of times an integration is updated, you can create an automation afaik